groupthink/commands.py

```diff
@@ -2,107 +2,129 @@
 import os
 import sys
 import argparse
+import argh
+import requests
+from argh import arg, aliases
 from glob import glob
 
-script_dir = '/usr/local/bin'
+dest = '/usr/local/bin'
 home = os.path.expanduser('~')
-repo_dir = '{}/.groupthink'.format(home)
-
-
-def install(org, alias=None, script_dir=script_dir, repo_dir=repo_dir):
-    # install organization scripts repository in ~/.groupthink
+storage = '{}/.groupthink'.format(home)
+
+@arg('group', help='The group to install')
+@arg('-a','--alias', help='Set a different name for this command')
+@arg('-d','--dest')
+@arg('-s', '--storage')
+def install(group, alias=None, dest=dest, storage=storage):
+    # install group scripts repository in ~/.groupthink
     # and add groupthink script to /usr/local/bin
-    alias = alias if alias else org
-    check_install(org, check_already=False, repo_dir=repo_dir)
-    script_exists = os.path.exists('{0}/{1}'.format(script_dir, org))
+    alias = alias if alias else group
+    check_install(alias, check_already=False, storage=storage)
```

Comment:
If people are installing the same command more than once under different names, I don't think we should stop them.